e2e/test/scenarios/custom-column/custom-column-reproductions.cy.spec.js

```diff
@@ -1415,3 +1415,81 @@ describe("issue 48562", () => {
     H.CustomExpressionEditor.value().should("contain", "[Unknown Metric]");
   });
 });
+
+describe("issue 54638", () => {
+  beforeEach(() => {
+    H.restore();
+    cy.signInAsNormalUser();
+    H.openOrdersTable({ mode: "notebook" });
+
+    // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
+    cy.findByText("Custom column").click();
+  });
+
+  it("should be possible to click documentation links in the expression editor help text popover (metabase#54638)", () => {
+    H.CustomExpressionEditor.type("case(");
+    H.CustomExpressionEditor.helpText().within(() => {
+      cy.findByText("Learn more")
+        .scrollIntoView()
+        .should("be.visible")
+        .then($a => {
+          expect($a).to.have.attr("target", "_blank");
+          // Update attr to open in same tab, since Cypress does not support
+          // testing in multiple tabs.
+          $a.attr("target", "_self");
```

Comment:
Instead of mocking the app we can mock the browser:

https://github.com/metabase/metabase/blob/621ea1feca4faaa51af4b1d56e3c81b195142121/e2e/test/scenarios/onboarding/home/browse.cy.spec.ts#L50-L66

Reply:
This only works in that test because the component is calling `window.open` explicitly AFAICT, we are just clicking a normal link here so it does not work.

Reply:
Right!
Then we can spy on `HTMLAnchorElement.prototype`: https://github.com/metabase/metabase/blob/926d2c617eb5db27336889e7dc71ec3e8f81fc97/e2e/test-component/scenarios/embedding-sdk/dashboard-click-behavior.cy.spec.tsx#L95-L151

Reply:
I think that test also uses a programmatic click (ie. metabase code calls `link.click()` manually), vs just allowing the browser to click the link. The spy does not get called when I test this locally.

Reply:
Thanks for checking!